server/serializer/confluence_server.go

```diff
@@ -181,9 +183,58 @@ func ConfluenceServerEventFromJSON(data io.Reader) (*ConfluenceServerEvent, erro
 		return nil, err
 	}
 
+	confluenceServerEvent.sanitizeURLs()
+
 	return &confluenceServerEvent, nil
 }
 
+func sanitizeURL(raw string) string {
+	if strings.TrimSpace(raw) == "" {
+		return raw
+	}
```

Comment:
I'm not entirely certain that we would want to return the raw string as-is here, maybe at this point we want to return an empty string so that we can make sure that no hyperlinks can be created with just whitespaces (since in theory it allows for that and it would bypass the checks in GetSpaceDisplayName or GetPageDisplayName)

Granted it's something that wouldn't really happen in a live environment but it might be good to guarantee it isn't a possibility 😄

Reply:
Good points! However, I think returning the raw string is the safer approach here. These would come from the Confluence webhook payload itself and i f Confluence sends whitespace as a URL that's already a data issue on their end. Returning empty could hide the problem while returning as is lets it flow through and be visible.
Also url.Parse() is very permissive and rarely fails. If it does fail returning the original is safer than silently dropping the URL which could break notifications entirely. The fix is specifically targeting the double-slash issue (//spaces/) which path.Clean handles correctly. Adding logging and empty-string returns adds complexity without clear benefit for the actual bug we're solving.

Reply:
Yeah totally fair point that if we ever get to a moment where confluence is sending faulty payloads the that degree there are bigger problems at hand 😄 I did wonder whether my comment would be reaching a bit out of scope but thought it better to ask for clarity anyways :D 

Thanks for the response!